.github/workflows/update-terraform-provider.yaml

```diff
@@ -0,0 +1,51 @@
+name: Update Terraform Provider
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  update-terraform-provider:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install jq
+        run: sudo apt-get install jq
+
+      - name: Check for New Terraform Provider Release
+        id: check_release
+        run: |
+          API_URL="https://api.github.com/repos/scaleway/terraform-provider-scaleway/releases/latest"
+          latest_release=$(curl -s $API_URL)
+          latest_version=$(echo $latest_release | jq -r '.tag_name' | sed 's/^v//')
```

Comment:
$ gh release list -R scaleway/terraform-provider-scaleway 
  and then find a way to limit the output?